aten/src/ATen/core/ivalue.h

```diff
@@ -543,6 +544,18 @@ struct TORCH_API IValue final {
     payload.u.as_int = i;
   }
 
+  IValue(c10::SymInt i) : tag(Tag::SymInt), is_intrusive_ptr(false) {
+    payload.u.as_int = i.data_;
+  }
+
+  bool isSymInt() const {
+    return Tag::SymInt == tag;
+  }
+
+  c10::SymInt toSymInt() const {
+    return c10::SymInt(payload.u.as_int);
+  }
+
```

Comment:
By the way, this means that stored IValues are a hazard for use-after-free for the "use coarse refcounting to figure out if SymInt is dead or alive". I don't think we store IValues very much, so should be ok